numpy/core/tests/test_dtype.py

```diff
@@ -856,8 +856,13 @@ class PackedStructure(ctypes.Structure):
             itemsize=18))
         self.check(PackedStructure, expected)
 
-    @pytest.mark.xfail(sys.byteorder != 'little',
-        reason="non-native endianness does not work - see gh-10533")
+    def test_complex_big_endian_structure_packed(self):
+        class BigEndStruct(ctypes.BigEndianStructure):
```

Comment:
Can you add a test with a packed LittleEndianStructure too?

Reply:
Added.